omnibus/config/software/datadog-agentless-scanner-finalize.rb

```diff
@@ -18,8 +18,16 @@
     license :project_license
 
     # Move system service files
+    mkdir "/etc/init"
+    move "#{install_dir}/scripts/datadog-agentless-scanner.conf", "/etc/init"
```

Comment:
nit: I'd add a trailing slash to ensure we don't attempt to create a file called `init`, but since the folder is created just before that's definitely not needed